tests/bcl-test/common-monotouch_corlib_xunit-test.dll.ignore

```diff
@@ -414,4 +414,7 @@ System.SpanTests.ReadOnlySpanTests.BinarySearch_Double
 System.Reflection.Tests.MethodBaseTests.Test_GetCurrentMethod_Inlineable
 
 # device issues, it was reported here: https://github.com/mono/mono/issues/14761
-System.Text.Tests.StringBuilderTests.Append_CharPointer_Null_ThrowsNullReferenceException
\ No newline at end of file
+System.Text.Tests.StringBuilderTests.Append_CharPointer_Null_ThrowsNullReferenceException
+
+# device failures, reported at: https://github.com/mono/mono/issues/17375
+System.Reflection.Tests.RuntimeReflectionExtensionsTests.GetRuntimeMethod
```

Comment:
```suggestion
Platform32:System.Reflection.Tests.RuntimeReflectionExtensionsTests.GetRuntimeMethod
```

Reply:
Also, looking at the logs, we also have failures on:

* System.Reflection.Tests.RuntimeReflectionExtensionsTests.GetRuntimeField
* System.Reflection.Tests.RuntimeReflectionExtensionsTests.GetRuntimeEvent
* System.Reflection.Tests.RuntimeReflectionExtensionsTests.GetRuntimeProperty

Reply:
```suggestion
Platform32:System.Reflection.Tests.RuntimeReflectionExtensionsTests.GetRuntimeMethod
Platform32:System.Reflection.Tests.RuntimeReflectionExtensionsTests.GetRuntimeField
Platform32:System.Reflection.Tests.RuntimeReflectionExtensionsTests.GetRuntimeEvent
Platform32:System.Reflection.Tests.RuntimeReflectionExtensionsTests.GetRuntimeProperty
```

Reply:
Thanks Manuel, tried to merge these ones but, @VincentDondain does not have allow commits from maintainers enabled.

Reply:
@mandel-macaque maybe you want to try

Reply:
Good catch, fixed